code/modules/projectiles/guns/projectile/osipr.dm

```diff
@@ -78,7 +78,7 @@
 		..()
 
 /obj/item/weapon/gun/osipr/attack_hand(mob/user)
-	if(((src == user.r_hand) || (src == user.l_hand)) && magazine)
+	if((user.held_items.Find(src)) && magazine)
```

Comment:
While it has nothing to do with the PR, why did Deity code this entire slowflake gun that just looks like a regular gun with lots of copypaste of regular gun code?